DQM/Integration/python/clients/onlinebeammonitor_dqm_sourceclient-live_cfg.py

```diff
@@ -126,7 +126,10 @@
 # BeamMonitor
 #-----------------------------
 process.dqmOnlineBeamMonitor = cms.EDProducer("OnlineBeamMonitor",
-MonitorName = cms.untracked.string("OnlineBeamMonitor")
+MonitorName         = cms.untracked.string("OnlineBeamMonitor"),
+AppendRunToFileName = cms.untracked.bool(False),
+WriteDIPAscii       = cms.untracked.bool(True),
+DIPFileName         = cms.untracked.string("BeamFitResultsForDIP.txt")
```

Comment:
Sorry Simone I forgot to add that the DIP filename should probably contain the full `/nfshome0` path, similar to
https://github.com/cms-sw/cmssw/blob/d1be6d93d0d1174f154c6374044146cbf5bc929c/DQM/Integration/python/clients/beamhlt_dqm_sourceclient-live_cfg.py#L128